src/backend/tests/teams.test.ts

```diff
@@ -354,7 +354,6 @@ describe('Teams', () => {
       const justiceLeagueUpdates = { ...justiceLeague, userArchivedId: 2, dateArchived: new Date() };
 
       vi.spyOn(prisma.team, 'update').mockResolvedValue(justiceLeagueUpdates);
-      const res = await TeamsService.archiveTeam(superman, justiceLeague.teamId);
```

Comment:
you should keep this line, so it actually tests the service method/ passes the test, just remove the `const res` so that the linting check also passes